### PR TITLE
Upgrade PatternFly dependencies

### PR DESCRIPTION
**Pull request by marshmalien**
##### SUMMARY
Should unblock [6164](https://github.com/ansible/awx/issues/6164)

------------

* patternfly v4.70.2
  * [release notes](https://github.com/patternfly/patternfly-next/blob/master/RELEASE-NOTES.md)
  * [@patternfly/patternfly](https://www.npmjs.com/package/@patternfly/patternfly)

* react-icons v4.7.22
  * [changelog](https://github.com/patternfly/patternfly-react/blob/master/packages/react-icons/CHANGELOG.md)
  * [@patternfly/react-icons](https://www.npmjs.com/package/@patternfly/react-icons)

* react-core v4.84.3
  * [release notes](https://github.com/patternfly/patternfly-react/blob/master/packages/react-docs/RELEASE-NOTES.md)
  * [changelog](https://github.com/patternfly/patternfly-react/blob/master/packages/react-core/CHANGELOG.md) 
  * [@patternfly/react-core](https://www.npmjs.com/package/@patternfly/react-core)

##### COMPONENT NAME

 - UI

### CODE REVIEW

File: awx/ui_next/src/screens/Template/Survey/SurveyPreviewModal.jsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ function SurveyPreviewModal({
                       type={`survey-preview-textArea-${q.variable}`}
                       value={q.default}
                       aria-label={i18n._(t`Text Area`)}
-                      disabled
+                      isDisabled
```
Comment (marshmalien) [inline awx/ui_next/src/screens/Template/Survey/SurveyPreviewModal.jsx:68]: Added `isDisabled` support to TextArea https://github.com/patternfly/patternfly-react/pull/5107

**Review by jakemcdermott — APPROVED** [review]

**Comment by jakemcdermott** [comment]
e2e :green_circle: , checked w/ full csp enabled and everything looks fine

**Comment by jakemcdermott** [comment]
There's two visual test failures: https://dashboard.cypress.io/projects/ui2mq7/runs/3608/overview

... but I'm also seeing the exact same test failures on other prs: https://dashboard.cypress.io/projects/ui2mq7/runs/3611/overview

So my conclusion is that the failures don't relate to this PR. The e2e are :green_circle: - merging

**Comment by unlikelyzero** [comment]
@jakemcdermott we were more interested in the changes not the failures

**Pull request merged**